lib/poxa/adapter/gproc.ex

```diff
@@ -49,18 +49,18 @@ defmodule Poxa.Adapter.GProc do
   def channels(pid \\ :_) do
     Ex2ms.fun do
       {{:p, :l, {:pusher, channel}}, ^pid, _} -> channel
-    end |> select |> Enum.uniq
+    end |> select  |> Enum.uniq
```

Comment:
Nit: Extra whitespace

Reply:
Nice catch. I should probably add `credo` after I finish this upgrade 😶